kinde_sdk/frontend/models/error.py

```
@@ -0,0 +1,90 @@
+    Do not edit the class manually.
+"""  # noqa: E501
+
+
+from __future__ import annotations
+import pprint
+import re  # noqa: F401
+import json
+
+from pydantic import BaseModel, ConfigDict, Field, StrictStr
+from typing import Any, ClassVar, Dict, List, Optional
+from typing import Optional, Set
```

Ruff (0.12.2)

Redefinition of unused `Optional` from line 22

Remove definition: `Optional`

(F811)